pkg/injector/patch.go

```diff
@@ -210,26 +210,26 @@ func updateAnnotation(target, add map[string]string, basePath string) []JSONPatc
 // This function will append a label to the pod, which points to the unique Envoy ID used in the
 // xDS certificate for that Envoy. This label will help xDS match the actual pod to the Envoy that
 // connects to xDS (with the certificate's CN matching this label).
-func updateLabels(pod *corev1.Pod, proxyUUID uuid.UUID) *JSONPatchOperation {
+func updateLabels(pod *corev1.Pod, label, value string) *JSONPatchOperation {
```

Comment:
Ideally this function signature should be the same as `updateAnnotation` given both are maps and need to be patched similarly.